tools/build_pytorch_libs.sh

```diff
@@ -177,7 +177,6 @@ function build() {
 		       -DCMAKE_INSTALL_LIBDIR="$INSTALL_DIR/lib" \
 		       -DCUDA_NVCC_FLAGS="$CUDA_NVCC_FLAGS" \
 		       -DCUDA_DEVICE_DEBUG=$CUDA_DEVICE_DEBUG \
-		       -DCMAKE_PREFIX_PATH="$INSTALL_DIR" \
```

Comment:
hooray!!!!
But this will break on machine with no system NCCL, because now setup.py builds NCCL, and cmake wont be able to find it.
If in case CMake is able to find it, something quite off is going on, i.e. NCCL (and gloo) is being built by both root cmake and setup.py invoking `build_pytorch_libs.sh` separately to build those libs.